GitHub Actions/haal-centraal-npm-beheer.md

```diff
@@ -160,6 +160,11 @@ Wil je met één statement alle packages updaten naar de meest recente versie, d
 - `ncu -u`. Met deze statement worden de npm package referenties in een package.json bestand ge-update naar de meest recente versie
 - `npm install`. Met deze statement worden de packages in de node_modules bestand geupdate aan de hand van het package.json bestand
 
+## Commit naar upstream repository
+
+Tijdens het hierboven beschreven proces wordt de folder 'node_modules' lokaal gegenereerd. Deze folder moet NIET gecommit worden naar de upstream repository. Committen naar de eigen fork van die repository wordt daarmee ook afgeraden omdat het dan erg lastig wordt de folder niet naar de upstream repository te committen.
+Je kunt voorkomen dat de betreffende folder gecommit wordt door hem in het '.gitignore' bestand te definiëren of hem simpelweg te verwijderen.
```

Comment:
`hem simpelweg te verwijderen` is geen goed idee. Als dit gebeurt vóór de commit, dan werkt het resolven en linten van de openapi.yaml niet meer tijdens de commit